megamek/src/megamek/server/Server.java

```diff
@@ -9956,6 +10022,64 @@ else if (bldgExited.equals(bldgEntered)
             }
         }
     }
+    
+    /**
+     * Updates the position of any towed trailers.
+     *
+     * @param tractor    The Entity that is moving
+     * @param trailer    The current trailer being updated
+     */
+    private void processTrailerMovement(Entity tractor, Entity trailer, ArrayList<Coords> trainPath) {
+        double trailerPositionOffset = 0;
```

Comment:
This should be an int.

Reply:
We agreed that this method has changed to the point that this one no longer applies.